dev-lua/lua-openssl/lua-openssl-0.4.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit toolchain-funcs
+
+DESCRIPTION="OpenSSL binding for Lua"
+HOMEPAGE="https://github.com/zhaozg/lua-openssl"
+SRC_URI="https://github.com/zhaozg/lua-openssl/archive/${PV}.tar.gz -> lua-openssl-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="luajit libressl"
```

Comment:
for libressl compatibility we will need a patch like https://patch-diff.githubusercontent.com/raw/zhaozg/lua-openssl/pull/81.patch but we should have upstream take a look over it, so remove libressl support for now